docs/platform_operators/external-databases.md

```diff
@@ -73,65 +72,129 @@ As prerequisite, you need to execute the following steps to configure your postg
     To use the Golang [yq](https://github.com/mikefarah/yq) utility, use these assignments.
     ```
     ```bash
-    CCDB_USERNAME=$(yq read "$VALUES_FILE" 'capi.database.user')
-    CCDB_PASSWORD=$(yq read "$VALUES_FILE" 'capi.database.password')
-    CCDB_NAME=$(yq read "$VALUES_FILE" 'capi.database.name')
-    UAADB_USERNAME=$(yq read "$VALUES_FILE" 'uaa.database.user')
-    UAADB_PASSWORD=$(yq read "$VALUES_FILE" 'uaa.database.password')
-    UAADB_NAME=$(yq read "$VALUES_FILE" 'uaa.database.name')
+    CCDB_USERNAME=$(yq read db-values.yml 'capi.database.user')
```

Comment:
how about using `"$DB_VALUES_FILE"` so that it can be set in a single place?

Reply:
OK